feg/protos/s8_proxy.proto

```diff
@@ -58,7 +58,17 @@ enum PDNType {
 }
 
 enum RATType {
```

Comment:
We should converge this RatType with the one used in session_manager.proto? (I prefer converging to this one since it seems like we're mapping to the actual value)